doc/use_cases.md

```diff
@@ -0,0 +1,49 @@
+# Use case scenarios for posteriordb
+
+posteriordb has different uses. Below is list of example use cases we have been thinking and what features are important in each use case
+
+## Testing implementations of inference algorithms with asymptotically decreasing bias and variance
+
+If an inference algorithm has asymptotically decreasing bias and variance, then an implementation should have decreasing error compared to true posterior summaries.
```

Comment:
it may be unclear to readers with what the algorithm is asymtotically decreasing. Also, it is unclear what "decreasing error compared to the true posterior summaries" means.

Reply:
Can you elaborate your comment? For someone testing algorithms asymptotic, bias and variance should be familiar. Do you think we should have glossary of these terms? Also decreasing, error and posterior summaries are well defined. Is there something strange in how I use these words?

Reply:
I think a problem here is asymptotically in what? In MCMC we would have the error go down with S, but in VB it would go down with increase optimization (if the posterior is in the variational family), in Laplace I do not really see that we have any asymptotics (rather than in the number of observations)?

Reply:
> I think a problem here is asymptotically in what? 

Compared to true. It explicitly separates the cases 1) "If an inference algorithm has asymptotically decreasing bias and variance" (like MCMC) and 2) doesn't (like VB and Laplace).

Reply:
Ok. I'll try to clarify this.